docs/getting-started/integrations/angular.md

```diff
@@ -1,14 +1,14 @@
 ---
 menu-title: Angular
 meta-title: Angular rich text editor component with npm | CKEditor 5 documentation
-meta-description: Install, integrate and configure CKEditor 5 using the Angular component with npm or ZIP.
+meta-description: Install, integrate and configure CKEditor 5 using the Angular component with npm.
 category: self-hosted
 order: 30
 ---
 
 {@snippet installation/integrations/framework-integration}
 
-# Angular rich text editor component
+# Angular rich text editor component with npm
```

Comment:
This is the 3rd time I look at this phrase `with npm` in this context, and something is wrong structurally.

(React and others affected)

`Angular rich text editor component (npm)`

`Angular rich text editor component -- setup with npm`

Reply:
Good point. I'll experiment with some titles and figure something out.

Reply:
I wanted to unify the title and h1. The title length is already close to the limit, so I used the shorter version.